columnq/src/table/mod.rs

```diff
@@ -137,7 +140,9 @@ pub enum TableLoadOption {
     },
     csv(TableOptionCsv),
     ndjson {},
-    parquet {},
+    parquet {
+        lazy: Option<bool>,
```

Comment:
We could use the same defaults pattern in the csv option struct to get rid of the option here, which should simplify downstream code by avoiding option match on every access.

Reply:
Modelled `TableOptionParquet` oafter `TableOptionCsv`, hope that fits the bill now.

Also renamed the parameter and inverted the meaning to `use_memory_table`, defaulting to true to maintain the current roapi behavior as the default. IMHO, this name better conveys the purpose of the configuration option than `lazy`.